src/HotChocolate/Core/src/Types.Scalars/HslaColorCodeType.cs

```diff
@@ -0,0 +1,114 @@
+using System.Text.RegularExpressions;
+using HotChocolate.Language;
+
+namespace HotChocolate.Types.Scalars
+{
+    /// <summary>
+    /// The `HslaColorCode` scalar type represents a valid a CSS HSLA color as defined
+    /// here https://developer.mozilla.org/en-US/docs/Web/CSS/color_value#hsl_colors.
+    /// </summary>
+    public class HslaColorCodeType : StringType
+    {
+        private static readonly string _validationPattern =
+            ScalarResources.HslColorCodeType_ValidationPattern;
```

Comment:
We can just leverage the HSL regex as it covers both

Reply:
Ah, in this case we can just extend the type I think
```
public class HslaColorCodeType : HslColorCodeType 
{
    ..ctors
}
```